From 728118680cbdd98d0197433b91794454afdffe2a Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Tue, 12 Jan 2016 18:21:20 +0800
Subject: [PATCH 0835/5242] MLK-12205 ARM: dts: 6sx-ard: add sensor node

commit  473e551bb7292c80adf10cbdeb5bf3c657759c3f from
https://source.codeaurora.org/external/imx/linux-imx.git

Add sensor node in 6sx-ard dts. The sensors are mma8451,
mag3110 and isl29023.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |   26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 9b876ab..a775c65 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -391,6 +391,32 @@
 		#gpio-cells = <2>;
 		resets = <&max7310_reset>;
 	};
+
+	mma8451@1c {
+		compatible = "fsl,mma8451";
+		reg = <0x1c>;
+		position = <7>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <24 8>;
+		interrupt-route = <1>;
+	};
+
+	mag3110@0e {
+		compatible = "fsl,mag3110";
+		reg = <0x0e>;
+		position = <2>;
+		interrupt-parent = <&gpio6>;
+		interrupts = <6 1>;
+	};
+
+	isl29023@44 {
+		compatible = "fsl,isl29023";
+		reg = <0x44>;
+		rext = <499>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <23 2>;
+	};
+
 };
 
 &lcdif2 {
-- 
1.7.9.5

